plugins/os/os_model.rb

```diff
@@ -188,11 +188,11 @@ def self.canonical_name
   end
 
   def self.buildable_systems
-    [OsOpenSuse13_2]
+    [OsOpenSuse13_2, OsOpenSuseTumbleweed, OsOpenSuseLeap]
```

Comment:
It will most likely work but might change in the nearby future because of the rolling release character so I don't want to risk it. I would even remove 13.2.